src/sentry/utils/performance_issues/detectors/n_plus_one_api_calls_detector.py

```diff
@@ -255,62 +249,11 @@ def _spans_are_concurrent(self, span_a: Span, span_b: Span) -> bool:
 
     def _spans_are_similar(self, span_a: Span, span_b: Span) -> bool:
         return (
-            self.span_hashes[span_a["span_id"]] == self.span_hashes[span_b["span_id"]]
+            parameterize_url(get_url_from_span(span_a))
+            == parameterize_url(get_url_from_span(span_b))
```

Comment:
These should be hashes, but I'm unsure what `span['hash']` is currently set to. Can I use that isntead, or are we not using parametized URLs for hashing spans yet?